fgl.cabal

```diff
@@ -67,6 +68,7 @@ library {
     build-depends:    base < 5
                     , transformers
                     , array
+                    , vector
```

Comment:
Which version(s) of `vector`?

Note that `vector` is not a boot library (like `base`, `containers` and `array`), and has only been shipped with the Haskell platform since `2012.4.0.0` (which had GHC-7.4.2); I want to be very cautious about adding dependencies that don't work with older versions of GHC in case someone, somewhere is still relying upon such a version.

(It did build on Travis-CI with `7.0.4`, so it's not a huge concern; but it's still probably a good idea to specify the version bounds just in case; technically speaking the other dependencies should have them as well but they're much more stable and less likely to change so I haven't touched them.)